app/services/assets/assets.go

```diff
@@ -129,6 +145,26 @@ func (a *Service) fetchEvmFungibleAssetInfo(networkId uint64, assetAddress strin
 	return assetInfo, err
 }
 
+func (a *Service) fetchEvmNonFungibleAssetInfo(networkId uint64, assetAddress string, evmTokenClients map[uint64]map[string]client.EvmNFT) (assetInfo assetModel.NonFungibleAssetInfo, err error) {
+
+	evmTokenClient := evmTokenClients[networkId][assetAddress]
```

Comment:
If there is a case where evmTokenClient can't be found we should use ok to check and return appropriate error

Reply:
There won't be such a case as this function is only used by the bridge config and we initialize the clients by the same config and this function is executed only on start.